desktop-exporter/app/routes/main-view.jsx

```diff
@@ -1,42 +1,73 @@
 import React from 'react';
-import { Link, useLoaderData } from "react-router-dom";
+import { Outlet, NavLink, useLoaderData } from "react-router-dom";
+import { FixedSizeList } from 'react-window';
+
 
 export async function mainLoader() {
     const response = await fetch("/api/traces");
     const traceSummaries = await response.json();
     return traceSummaries;
 }
 
-export default function MainView() {
+
+function Row({ index, style, data }) {
+    return (
+        <NavLink to={`traces/${data[index].traceID}`} style={style}>
+            {data[index].traceID}
+        </NavLink>
+    );
+}
+
+function useToggle(initialValue = false) {
+    const [value, setValue] = React.useState(initialValue);
+    const toggle = React.useCallback(() => {
+        setValue((v) => !v);
+    }, []);
+    return [value, toggle];
+}
```

Comment:
suggestion: We can pull this in from this library since we're going to want `useElementSize` from it later: https://usehooks-ts.com/react-hook/use-toggle

```js
import { useToggle } from 'usehooks-ts'
```

Reply:
It works!